src/lib/list/list.scss

```diff
@@ -262,6 +262,22 @@ $mat-list-item-inset-divider-offset: 72px;
   }
 }
 
+mat-action-list {
+  //remove the native button look and make it look like a list item
+  button {
```

Comment:
Shouldn't this apply to all `button`s for any `.mat-list-item` not just the ones under a `mat-action-list`?

Reply:
As we talked,  we added mat-action-list specifically for action list to make it more clear, so the idea is to let user use it under mat-action-list.